third_party/bigframes_vendored/pandas/core/series.py

```diff
@@ -2154,3 +2207,30 @@ def iat(self):
     def at(self):
         """Access a single value for a row/column label pair."""
         raise NotImplementedError(constants.ABSTRACT_METHOD_ERROR_MESSAGE)
+
+    @property
+    def values(self):
+        """
+        Return Series as ndarray or ndarray-like depending on the dtype.
+
+        .. warning::
+           We recommend using :attr:`Series.array` or
+           :meth:`Series.to_numpy`, depending on whether you need
+           a reference to the underlying data or a NumPy array.
```

Comment:
This isn't really true for bigframes - the underlying memory is in bigquery

Reply:
Thanks, also realized we don't implement `array` yet. Removed this part, PTAL